app/Http/Controllers/RecommendationController.php

```diff
@@ -198,42 +208,58 @@ public function updateAdmin($input)
 
     public function updateUserRec($input)
     {
-        // These aren't the real rules, it's just needed to call the array validation class
-    $rules = ['first_name' => 'required|TextFieldArray'];
-        $data = ['first_name' => $input['rec']];
-    // Calls the class that goes through and checks the real rules.
-    $v = Validator::make($data, $rules);
-
-        if ($v->fails()) {
-            return  redirect()->back()->with('flash_message', ['text' => 'All fields are required.', 'class' => '-error'])->withInput();
+      $rules = ['first_name'      => 'required', 
```

Comment:
It looks like in other places ([voting app](https://github.com/DoSomething/voting-app/blob/42bddbd4bdf69a14b065b0dedcdde15937f702ba/app/Http/Controllers/CandidatesController.php#L18-L23)) validation rules live in the controller? I should pull them out of this function anyway and call it `$user_rules` or something like that and call it on `store` too because it turns out that isn't actually validating currently